backend/src/modules/authorization/router.js

```diff
@@ -64,12 +63,31 @@ router.post('/signUp', checkSchema({
   const errors = validationResult(req)
 
   if (!errors.isEmpty()) {
-    return res.json(errors.array())
+    return res.json({ errors: errors.array() })
   }
 
   const result = await Controller.signUp(req.body)
 
   return res.json(result)
 })
 
+router.post('/logOut', checkSchema({
+  authorization: {
+    in: 'body',
+    notEmpty: {
+      errorMessage: 'token is missing',
+    },
+  },
+}), async (req, res) => {
+  const errors = validationResult(req)
+  if (!errors.isEmpty()) {
+    res.json({ errors: errors.array() })
+  }
+  const bool = await Controller.logOut(req.body)
+  if (bool) res.status(200)
+  else res.status(500)
+  res.json({ logout: bool })
+  return bool
```

Comment:
Again, I cannot read this part, 'cause it's unreadable. Use some line breaks, please :)